### PR TITLE
raidboss: Update Titan Unreal Rock Throw

### LINKED ISSUE #2172: [Suggestion] NetRegexes.tether({ id: '0007' }) for Rock Throw
https://github.com/quisquous/cactbot/blob/ad2d3a9857ad1d1445b73b68ae4d995be170d2be/ui/raidboss/data/05-shb/trial/titan-un.js#L66

First off, I apologize if this is the wrong way to submit a suggestion/fix.

There is a **23: NetworkTether** line (ID: 0007) before the **15: NetworkAbility** line for Rock Throw. The DPS is listed as the source while the healer is listed as the target.

> `[23:30:52.184] 23:1025775F:Riku Tribal:102D78FF:Mahiru Hoshino:0000:0000:0007:102D78FF:000F:0000:`
> `[23:30:52.453] 15:400188BC:Titan:5ADD:Rock Throw:1025775F:Riku Tribal:`

> `[23:32:06.763] 23:1025775F:Riku Tribal:102D78FF:Mahiru Hoshino:6D20:0000:0007:102D78FF:000F:2020203A:`
> `[23:32:06.985] 23:1025775F:Riku Tribal:102D78FF:Mahiru Hoshino:B330:0000:0007:102D78FF:000F:0000:`
> `[23:32:06.985] 15:400188BC:Titan:5ADD:Rock Throw:1025775F:Riku Tribal:`

> `[23:32:51.895] 23:101F5B64:Knyssa Callistis:1033BE65:Ankey Kilth:42C0:0000:0007:1033BE65:000F:3E4CCCCD:`
> `[23:32:52.163] 15:400188BC:Titan:5ADD:Rock Throw:101F5B64:Knyssa Callistis:`

> `[23:34:34.252] 23:101F5B64:Knyssa Callistis:102D78FF:Mahiru Hoshino:0000:0000:0007:102D78FF:000F:0000:`
> `[23:34:34.517] 15:400188BC:Titan:5ADD:Rock Throw:101F5B64:Knyssa Callistis:`

> `[23:37:13.840] 23:1025775F:Riku Tribal:102D78FF:Mahiru Hoshino:6D20:0000:0007:102D78FF:000F:2020203A:`
> `[23:37:14.018] 15:400188BC:Titan:5ADD:Rock Throw:1025775F:Riku Tribal:`

### PR DESCRIPTION
**Pull request by panicstevenson**
Update Rock Throw triggers to use tethers intead, giving an alert to
people with the jail tether, and a mild reminder for everyone else.

Closes #2172

### CODE REVIEW

File: ui/raidboss/data/05-shb/trial/titan-un.js [hunk 1]
```diff
@@ -64,22 +64,20 @@ export default {
...
+          return output.jailOnYou();
+      },
+      infoText: (data, matches, output) => {
+        if (matches.source !== data.me && matches.target !== data.me)
+          return output.jails();
+      },
       outputStrings: {
-        text: {
+        jailOnYou: {
+          en: 'Jail on YOU',
```
Comment (VanSlanzar) [inline ui/raidboss/data/05-shb/trial/titan-un.js:78]: `fr: 'Geôle sur VOUS',`
Hi, if possible to add this. ty.

File: ui/raidboss/data/05-shb/trial/titan-un.js [hunk 2]
```diff
@@ -64,22 +64,21 @@ export default {
...
-      netRegexKo: NetRegexes.ability({ source: '타이탄', id: '5ADD', capture: false }),
-      // This trigger is a little tricky.  Ideally we'd want to alert all
-      // players who had a jail marker, but this is not a 1B headmarker.
-      // This is probably because this is old content, similar to T5.
-      // Worse, only one player is called out as the target of Rock Throw.
-      // It seems strictly worse to differentiate trigger output of known
-      // rock throw target vs unknown rock throw target, as we don't want to
-      // give a false expectation that this trigger is very smart.
-      alertText: (data, _, output) => output.text(),
+      netRegex: NetRegexes.tether({ id: '0007' }),
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/trial/titan-un.js:67]: Aren't there two jails? If so, will this call out the jail info twice?

(Also can you update titan extreme similarly? I have logs if you need them.)

Comment (panicstevenson) [reply ui/raidboss/data/05-shb/trial/titan-un.js:67]: So, it's one tether log, with one player as source and the other as target. And yes please to logs!

Comment (quisquous) [reply ui/raidboss/data/05-shb/trial/titan-un.js:67]: Oh in the example issue it looks like there's one with two tethers? Should there be a suppressSeconds then? If you've run this, do you see that happening anywhere?

**Pull request merged**